src/app/templates/client/Client.jsx

```diff
@@ -123,10 +145,14 @@ function Client() {
       onDragLeave={handleDragLeave}
       onDrop={handleDrop}
     >
-      <div className="navigation__wrapper">
+      <div className={`navigation__wrapper
+        ${activeView !== viewPossibilities.nav ? 'client-container-item__hidden' : null}`}
+      >
         <Navigation />
       </div>
-      <div className="room__wrapper">
+      <div className={`room__wrapper
+        ${activeView !== viewPossibilities.room ? 'client-container-item__hidden' : null}`}
+      >
```

Comment:
Unnecessary line-breaks are added to the className. Also null is added to the className if condition is false. 

I think using `ref` to change there visibility will be more good instead of re-rendering this component which is very costly. Also with ref we can also add nice animation to make it feel more smooth.